### DISCUSSION TITLE
Camera Storage

### DISCUSSION THREAD

**Discussion by ThorstenUm** [blakeblackshear/frigate#12593] [General]
Hello!
I use the 0.14 beta 4. How is the camera storage that is displayed on the Storage tab calculated?
"Recordings" is displayed as 786GiB. The 4 existing cameras are shown as 60.4GiB each.

Thanks

**Comment by NickM-27** [comment]
It is calculated from the DB, meaning there are recordings retained that are not assigned to those 4 cameras. A screenshot would be helpful to see

**Reply by ThorstenUm** [reply]
attached is the screenshot
![Screenshot 2024-07-24 181222](https://github.com/user-attachments/assets/3815cd7b-65de-449d-8b61-4da30d1204f4)

**Reply by NickM-27** [reply]
I think you misunderstand, 700GB is the total size available, 61GB is how much is being used

**Reply by ThorstenUm** [reply]
ah ok, this is the proportional ratio among all cameras.
Thanks

**Comment by dgriswo** [comment]
I found this thread while looking to see where I could adjust the "Camera Storage" limit because it was at 100%, or so I thought.

I feel like the UI is not intuitive.  The Overview/Recordings graphic is the total disk consumed by all cameras vs total storage capacity.  The Camera Storage/${camera_name} graphic is the percentage that ${camera_name} is consuming vs total disk consumed by all cameras..

![image](https://github.com/user-attachments/assets/84740a33-5766-471b-bf23-49bd93de2cf3)

**Reply by NickM-27** [reply]
it's already been changed for the next version 
![Screen Shot 2024-09-06 at 12 07 12 PM](https://github.com/user-attachments/assets/d9419d2b-23c9-4c5c-b1f2-6a2d6f197734)

**Reply by dgriswo** [reply]
That looks really good!  Much more intuitive.

**Reply by adambonny** [reply]
Brilliant! Can here with the same confusion and there’s already a sexy dashboard update coming 👍🏼

**Reply by kade-d** [reply]
Same here. This looks awesome! 🥂

**Reply by Wolk9** [reply]
That looks like a tremendous good change!
